src/Nordea/Components/MultiSelectDropdown.elm

```diff
@@ -0,0 +1,250 @@
+                        , css
+                            [ maxWidth (pct 100)
+                            , cursor pointer
+                            , hover [ backgroundColor Colors.coolGray ]
+                            , displayFlex
+                            , flexDirection row
+                            , padding (rem 0.5)
+                            , alignItems center
+                            , Css.property "gap" "0.5rem"
+                            ]
+                        ]
+                        [ Checkbox.init option.name (Html.text option.label) option.onCheck
+                            |> Checkbox.withIsChecked option.isChecked
+                            |> Checkbox.withAppearance Checkbox.Simple
+                            |> Checkbox.view []
+                        ]
+
+                dropdownStyles =
+                    Css.batch
+                        [ backgroundColor Colors.white
+                        , borderBottom3 (rem 0.0625) solid Colors.grayMedium
+                        , borderLeft3 (rem 0.0625) solid Colors.grayMedium
+                        , borderRight3 (rem 0.0625) solid Colors.grayMedium
+                        , borderBottomLeftRadius (rem 0.25)
+                        , borderBottomRightRadius (rem 0.25)
+                        , boxSizing borderBox
+                        , padding3 (rem 0.5) (rem 0) (rem 0.0)
+                        ]
+            in
+            Html.div
+                [ css
+                    [ displayFlex
+                    , flexDirection column
+                    , position absolute
+                    , top (pct 100)
+                    , left (rem 0)
+                    , right (rem 0)
+                    , if dropdown.hasFocus then
+                        display block
+
+                      else
+                        display none
+                    ]
+                ]
+                [ Html.ul
+                    [ css
+                        [ overflowY scroll
+                        , maxHeight (rem 16.75)
+                        , dropdownStyles
+                        , listStyle none
+                        ]
+                    ]
+                    (dropdown.options |> List.map viewOption)
+                ]
+    in
+    Html.fieldset ([ css [ displayFlex, flexDirection column ] ] ++ attrs)
+        [ Html.row [ css [ displayFlex, justifyContent spaceBetween, marginBottom (rem 0.2) ] ]
+            [ Label.init { label = dropdown.label } |> Label.withAsLegend |> Label.view []
+            , dropdown.requirednessHint |> Html.viewMaybe RequirednessHint.view
+            ]
+        , Html.div
+            [ class "nordea-select"
+            , Events.on "focusout" (Decode.succeed (dropdown.onFocus False))
+            , Events.on "focusin" (Decode.succeed (dropdown.onFocus True))
+            , css [ position relative ]
+            , tabindex -1
```

Comment:
I think these should be moved up to the root element. The expected behaviour for clicking labels is to transfer the focus to the form control. What do you think?

Reply:
Also, I was unable to trigger the dropdown with just my keyboard, I think it may because of the `tabindex -1`. I think setting it to 0 is the way to go:


_tabindex="0" means that the element should be focusable in sequential keyboard navigation, after any positive tabindex values and its order is defined by the document's source order._

Reply:
Nice! That fixed it🙏 Not able to select or deselect checkboxes though, but that seems to be a problem with that component

Reply:
Ah yes, just tested in PartnerHub. Tabbing checkboxes and radio buttons work, and you can toggle the state with space (as expected), but the focus outline doesn't show; making it look like it doesnt work. Thats a bug